lib/storekit/receipt.rb

```diff
@@ -1,19 +1,42 @@
+require 'ostruct'
+
 module StoreKit
-  class Receipt
-    attr_reader :receipt_data
+  class Receipt < OpenStruct
+    TRANSFORM = {
+       'unique_identifier' => true,
+       'product_id' => true,
+       'bid' => true,
+       'unique_vendor_identifier' => true,
+       'bvrs' => true,
+       'transaction_id' => proc { |v| v.to_i },
+       'original_transaction_id' => proc { |v| v.to_i },
+       'quantity' => proc { |v| v.to_i },
+       'item_id' => proc { |v| v.to_i },
+       'web_order_line_item_id' => proc { |v| v.to_i },
```

Comment:
The documentation is clear when strings are to be given special interpretation, eg. like in the time fields, and I quote:

> string, interpreted as an RFC 3339 date

While it is true that at the time of writing, the IDs can be parsed as integers, there is no guarantee that this will always be the case. If Apple decides to start adding alphabets into the IDs, the conversion to integer will fail. This will then result in having to do an emergency patch on our production apps.